### PR TITLE
Add James's directory and chapter 1

### PR DESCRIPTION
**Pull request by JamesHeo**
This pull request is for adding James's directory and confluence links for chapter 1

### CODE REVIEW

**Comment by wjrmffldrhrl** [comment]
I have no permission to access the links 😭 
Screenshot 2022-12-02 at 10 53 59 PM

**Comment by JamesHeo** [comment]
My fault. Could you try again, please?

**Comment by wjrmffldrhrl** [comment]
It still does not work  

You don't have to resolve this now, don't hurry

**Comment by wjrmffldrhrl** [comment]
Screenshot 2022-12-02 at 11 16 48 PM
I can see this page when I access that links with my company account

File: personal/james/chapter1/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[confluence-links]
+	korean : https://james-heo.atlassian.net/wiki/spaces/LN/pages/33200
+	english : https://james-heo.atlassian.net/wiki/spaces/LN/pages/65545/Why+Terraform
```
Comment (wjrmffldrhrl) [inline personal/james/chapter1/README.md:3]:
```suggestion
# confluence-links
- korean : https://james-heo.atlassian.net/wiki/spaces/LN/pages/33200
- english : https://james-heo.atlassian.net/wiki/spaces/LN/pages/65545/Why+Terraform
```

**Review by wjrmffldrhrl — COMMENTED** [review]
Edit text formatting to markdown

File: personal/james/chapter1/README.md (hunk 1 above)
Comment (JamesHeo) [reply personal/james/chapter1/README.md:3]: Thank you :)

Comment (wjrmffldrhrl) [reply personal/james/chapter1/README.md:3]: If you want to apply my suggestion, Click the `Commit suggestion` button and add a commit  

However, this is just a "suggestion", so you don't have to 😄

**Comment by JamesHeo** [comment]
> 



> Edit text formatting to markdown

**Comment by JamesHeo** [comment]
Woops I have accidentally closed it lol.
I just wanted to say thank you for letting me know how to apply your changes into my branch. 

What

**Comment by wjrmffldrhrl** [comment]
It still can't access. but, you can release the permission after 😄

**Comment by JamesHeo** [comment]
Thanks!
Yep, I am aware. I actually plan to convert them to markdown format this week.
I will send another PR this week. 

Thank you for looking into it together! :D

**Pull request merged**